misc/python/materialize/release/auto_cut_release.py

```diff
@@ -11,13 +11,13 @@
 
 import os
 import sys
-from pathlib import Path
 from datetime import datetime, timedelta
+from pathlib import Path
```

Comment:
I'm confused, you fixed this already in https://github.com/MaterializeInc/materialize/commit/360f19f3d8b833f5270a3bdcf6349323e827cb43, not sure why it's showing up as a diff in this PR too